server/datastore/mysql/migrations/tables/20210601000008_TeamsEnrollSecrets.go

```diff
@@ -71,6 +71,7 @@ func Up_20210601000008(tx *sql.Tx) error {
 		ALTER TABLE enroll_secrets
 		DROP COLUMN active,
 		DROP COLUMN name,
+		ADD PRIMARY KEY (secret),
```

Comment:
You should run `make dump-test-schema` to make sure tests run with the latest schema.